### PR TITLE
Update dynamic-theme-fixes.config

### PR DESCRIPTION
**Pull request by cinderblox94**
Added my web to the dynamic

### CODE REVIEW

**Comment by Myshor** [comment]
Can not check the fixes because I can open the site.

**Comment by cinderblox94** [comment]
Yes, it's an internal enterprise web. But i cannot keep saved the changes
when i close firefox
Álvaro Cañizares

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -14223,6 +14223,26 @@ INVERT
 svg[class$="logo-image"]
 .explorer__header-close
 
+================================
+knossos.sice.com
```
Comment (Myshor) [inline src/config/dynamic-theme-fixes.config:14227]: missing empty line before URL

That breaks tests.

**Review by Myshor — CHANGES_REQUESTED** [review]
Need changes.

**Comment by Myshor** [comment]
After fixing code can you at least share screensots before/after with parts included in the fix?

**Comment by cinderblox94** [comment]
> Updated screeshots with/out the fixes

image
image

**Review by Myshor — APPROVED** [review]
Approved

**Pull request merged**